internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -830,6 +830,25 @@ func convertDeploymentConfig(in manifest.DeploymentConfiguration) template.Deplo
 	return out
 }
 
+func convertWorkerDeploymentConfig(in manifest.WorkerDeploymentConfig) template.DeploymentConfigurationOpts {
+	out := template.DeploymentConfigurationOpts{
+		MinHealthyPercent: minHealthyPercentDefault,
+		MaxPercent: maxPercentDefault,
```

Comment:
Why is this `MaxPercent` instead of `MaxHealthyPercent`?

Reply:
I was just following the example of the existing code. @paragbhingre??

Reply:
He was following this: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-service-deploymentconfiguration.html